routes.js

```diff
@@ -21,11 +21,38 @@ async function getForecast(request, response) {
   response.status(200).send(forecastResponse.data);
 }
 
+async function getYelp(request, response) {
+  let searchQuery = request.query.searchQuery;
+  let lat = request.query.latitude;
+  let lon = request.query.longitude;
+
+  console.log(searchQuery, lat, lon, '<---- YELP SEARCH QUERY LOG');
+
+  // const yelpURL = `https://api.yelp.com/v3/businesses/search?latitude=${lat}&longitude=${lon}`;
+
+
+
+
+
+  https://api.yelp.com/v3/businesses/search?=latitude=47.6038321&longitude=-122.3300624
+
+
+
+
+
+  // console.log(yelpURL, '<---- YELP URL LOG ---<<<');
+
+  // const yelpResponse = await axios.get(yelpURL);
+
+  // console.log(yelpResponse, '<---- YELP RESPONSE LOG ---<<<');
+
```

Comment:
Really nice work here!

When you can, go through the files and clean up unused code. It is best practice to not commit unused code 👍